app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -84,4 +84,11 @@ def vm_restart(vm, _options = {})
   def self.display_name(number = 1)
     n_('Cloud Provider (VMware vCloud)', 'Cloud Providers (VMware vCloud)', number)
   end
+
+  def vm_create_snapshot(vm, options = {})
+    options[:quiesce] = 'false'
+    with_provider_connection do |service|
+      service.post_create_snapshot(vm.uid_ems, options)
```

Comment:
You can simply use `ems_ref` instead of `vm.uid_ems`, because we're on Cloud Provider anyway.

Reply:
Also, this seems to be asynchronous task but we usually wait for tasks to complete. So could you please go with:

```
response = service.post_create_snapshot(ems_ref, options)
service.process_task(response.body)
```

(see [here](https://github.com/ManageIQ/manageiq-providers-vmware/pull/184/files#diff-31414d950868b1eb5e1e513e2cb232a2R16))